.github/workflows/test-pr.yml

```diff
@@ -81,31 +81,25 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: recursive
-      - name: 'Set up Docker'
-        uses: ./.github/actions/with-docker
-        with:
-          tag: k-ci
-          distro: jammy
-          llvm: 14
-      - name: 'Build and Test K'
-        run: docker exec -t k-ci /bin/bash -c 'mvn verify --batch-mode -U'
       - name: 'Check out k-exercises'
         uses: actions/checkout@v3
         with:
           repository: kframework/k-exercises
           token: ${{ secrets.JENKINS_GITHUB_PAT }}
           submodules: recursive
           path: k-exercises
-      - name: 'Tutorial Integration Tests'
-        run: |
-          docker exec -t k-ci /bin/bash -c 'k-distribution/target/release/k/bin/spawn-kserver kserver.log'
-          docker exec -t k-ci /bin/bash -c 'cd k-exercises/tutorial && make -j`nproc` --output-sync'
-          docker exec -t k-ci /bin/bash -c 'cd k-distribution/k-tutorial/1_basic && ./test_kompile.sh'
-      - name: 'Tear down Docker'
-        if: always()
-        run: |
-          docker stop --time=0 k-ci
-          docker container rm --force k-ci || true
+      - name: 'Run K Tests'
+        uses: ./.github/actions/with-docker
+        with:
+          tag: k-ci
+          distro: jammy
+          llvm: 14
+          script: |
+            |
```

Comment:
What does the double pipe for this?

Reply:
It's to avoid YAML parsing errors I guess: https://github.com/octokit/request-action/issues/6